requirements-dev.txt

```
@@ -19,6 +19,7 @@ qiskit-aer>=0.15.0; python_version < "3.13"
 qiskit-qir
 qiskit-ionq>=0.5.12
 pyqpanda3>=0.2.0; python_version < "3.13"
+autoqasm
```

```suggestion
autoqasm>=0.1.2,<0.2
```

Set upper bound to protect from breaking changes